src/os/filestore/FileStore.cc

```diff
@@ -1742,17 +1742,17 @@ int FileStore::mount()
   if (!m_disable_wbthrottle) {
     wbthrottle.start();
   } else {
-    dout(0) << "mount INFO: WbThrottle is disabled" << dendl;
+    dout(0) << __func__ << " INFO: WbThrottle is disabled" << dendl;
     if (cct->_conf->filestore_odsync_write) {
-      dout(0) << "mount INFO: O_DSYNC write is enabled" << dendl;
+      dout(0) << __func__ << " O_DSYNC write is enabled" << dendl;
```

Comment:
maybe we can keep `INFO` to be backward compatible, unless we have a good reason not to do so?